index.html

```diff
@@ -228,5 +228,37 @@ <h3>Languages</h3>
             </div>
         </div>
     </div>
-</body>
-</html>
\ No newline at end of file
+    <form class="former-block">
+                <div class="form-block">
+                    <section>
```

Comment:
- [ ] It's better if the page had a `100vw` width and all the content would be centered using flexbox for example. This way your website would look great in all mobile screen sizes and it would match the design in Figma (in `375px` screen size you have empty spaces on the left and on the right.
- [ ] Kindly add some space between the previous section (about me) and this section (contact) 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/200881440-8ec25c05-ffe2-497e-ae80-67a9d767708e.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/200881494-422e4089-c595-4fe2-a0d9-036ee1b44253.png)